src/providers/hana/qgshanaconnectionstringbuilder.cpp

```diff
@@ -15,48 +15,105 @@
  *
  ***************************************************************************/
 #include "qgshanaconnectionstringbuilder.h"
+#include <QRegExp>
```

Comment:
QRegExp is deprecated and removed in qt 6 -- this should be QRegularExpression instead.

On that note it would be good to enable Hana on the qt6 ci builds to catch things like this 👍